src/FakeItEasy/Configuration/ArgumentCollection.cs

```diff
@@ -54,7 +53,7 @@ public int Count
         /// <summary>
         ///   Gets the names of the arguments in the list.

```

Comment:
Ugh. There's an opportunity to add docs here around when the names might be null. I dropped the ball. Stay tuned.